drivers/usb/typec/tcpm/fusb302.c

```diff
@@ -1740,6 +1773,11 @@ static int fusb302_probe(struct i2c_client *client,
 	init_tcpc_dev(&chip->tcpc_dev);
 	fusb302_debugfs_init(chip);
 
+	chip->gpio_sel = devm_gpiod_get(dev, "sel", GPIOD_OUT_HIGH);
```

Comment:
```suggestion
	chip->gpio_sel = devm_gpiod_get_optional(dev, "sel", GPIOD_OUT_HIGH);
```
语义上可能更适当，这样当没有定义sel时会返回NULL，只有在其他错误时会返回ERR